wao/strategy_controller.py

```diff
@@ -12,7 +12,7 @@
 class StrategyController:
     romeo_pool = {}
 
```

Comment:
have a local variable here : brain
and set it in setup()
self.brain = brain

then use it everywhere in this class and remove the "brain" parameters passed in the 4 methods below setup()

you can use self.brain instead